pkg/controllers/dynakube/oneagent/daemonset/env_vars_test.go

```diff
@@ -284,3 +287,30 @@ func assertReadOnlyEnv(t *testing.T, envs []corev1.EnvVar) {
 	assert.Equal(t, env.Name, oneagentReadOnlyMode)
 	assert.Equal(t, "true", env.Value)
 }
+
+func TestIsProxyAsEnvVarDeprecated(t *testing.T) {
+	t.Run("empty version", func(t *testing.T) {
+		got, err := IsProxyAsEnvVarDeprecated("")
+		assert.Nil(t, err)
+		assert.Equal(t, false, got)
+	})
+	t.Run("wrong format", func(t *testing.T) {
+		_, err := IsProxyAsEnvVarDeprecated("1.2")
+		assert.NotNil(t, err)
+	})
+	t.Run("older version", func(t *testing.T) {
```

Comment:
nitpick: I would group similar tests into data driven approach to reduce lines of code

Reply:
<img src="https://media2.giphy.com/media/YS69e4g1D6360y6Aqv/giphy.gif"/>

Reply:
Fixed @andriisoldatenko